JitsiConferenceEvents.ts

```diff
@@ -223,22 +223,6 @@ export enum JitsiConferenceEvents {
      */
     PRIVATE_MESSAGE_RECEIVED = 'conference.privateMessageReceived',
 
-    /**
-     * Event fired when JVB sends notification about interrupted/restored user's
-     * ICE connection status or we detect local problem with the video track.
-     * First argument is the ID of the participant and
-     * the seconds is a string indicating if the connection is currently
-     * - active - the connection is active
-     * - inactive - the connection is inactive, was intentionally interrupted by
-     * the bridge
-     * - interrupted - a network problem occurred
-     * - restoring - the connection was inactive and is restoring now
-     *
-     * The current status value can be obtained by calling
-     * JitsiParticipant.getConnectionStatus().
-     */
-    PARTICIPANT_CONN_STATUS_CHANGED = 'conference.participant_conn_status_changed',
```

Comment:
Is there a replacement for this?

Reply:
In the new signaling mode, connectivity is determined per source (remote track) basis and JitsiTrackEvents.TRACK_STREAMING_STATUS_CHANGED is fired.